recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -40,6 +40,10 @@ do_install_append_class-target () {
 do_install_append_class-native () {
     rm -f ${D}${bindir}/aktualizr
     rm -f ${D}${bindir}/aktualizr-info
+    install -d ${D}${libdir}/sota
+    install -m 0644 ${S}/config/sota_autoprov.toml ${D}/${libdir}/sota/sota_autoprov.toml
+    install -m 0644 ${S}/config/sota_hsm_test.toml ${D}/${libdir}/sota/sota_hsm_test.toml
+    install -m 0644 ${S}/config/sota_implicit_prov.toml ${D}/${libdir}/sota/sota_implicit_prov.toml
```

Comment:
This means all of the important configs will be installed on every device, right? I think I actually like that.

Reply:
Um... No, that's actually only for native installation. They are later used by provisioning recipes. But we might have this option as well.

Reply:
Ah, now I see. I read too fast and didn't see the context that that was only in the `-native` part.